### ISSUE TITLE
Vault economy balance bug

### ISSUE THREAD

**Issue by FrankHeijden** [RockinChaos/ItemJoin#113]
https://pastebin.com/fpwfQi3Q

this item:

```
  serverselector:
    id: COMPASS
    slot: 0
    name: '&b&k||&3&lServer Selector&b&k||'
    lore:
      - '&7Click me to open the server selector!'
    commands:
      multi-click:
        - 'player: shop'
    commands-type: interact, inventory
    commands-sound: BLOCK_NOTE_PLING
    commands-cost: 0
    commands-cooldown: 5
    cooldown-message: '&9&lAquaCraft &7» &cYou can''t use this item for another &4%timeleft% &cseconds!'
    itemflags: unbreakable
    triggers: join, respawn, world-change, region-enter
    permission-node: ''
    enabled-regions: __global__
    enabled-worlds: world, world_nether, world_the_end
```

it occures when you fill in a thingy with "0". e.g.     "commands-cost: 0"

**Comment by RockinChaos** [comment]
You can set vault to false or completely remove the commands-cost from the items and it wont display this error. I will be releasing a fix for this in the next day or two.

**Comment by RockinChaos** [comment]
Issue is fixed in v4.2 you can get it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=171839
